react-context/client/src/pages/HomePage.js

```diff
@@ -0,0 +1,27 @@
+import { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import AppContext from '../components/AppContext';
+import './HomePage.css';
+
+export default function Home() {
+  /* TODO: Grab `user` from `AppContext` */
+  const user = useContext(AppContext);
```

Comment:
This doesn't work right. `useContext` returns an object and you need to destructure the `user` from that object. The way this code is currently written, when the user logs out they will not be automatically redirected to the sign-in page. If written properly, they will be.